src/System.Management.Automation/engine/SessionStateContainer.cs

```diff
@@ -2382,16 +2382,38 @@ private bool HasGetChildItemDynamicParameters(ProviderInfo providerInfo)
                         // Since the path contained glob characters or we are recursing and the
```

Comment:
@iSazonov, your last commit had 5 failures in `PowerShell-CI-linux`
Extended1.Get-ChildItem -Path -Depth 2 -Name -Force !!!!!!!!
```powershell
Expected 2, but got 1.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 83
83:         1 | Should -Be 2
```
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path -Depth -Name.Get-ChildItem -Path -Depth 2 -Name -Force
```powershell
Expected 13, but got 12.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 444
444:             $result.Count | Should -Be 13
```
Get-Process.Test for process property = Name
```powershell
Expected strings to be the same, but they were different.
Expected length: 4
Actual length:   15
Strings differ at index 0.
Expected: 'pwsh'
But was:  'ConsoleHost mai'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 74
74:         (Get-Process -Id $PID).Name | Should -BeExactly "pwsh"
```
Process Parent property.Has Parent process property
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 128
128:         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent' | Should -Not -BeNullOrEmpty
```
Process Parent property.Has valid parent process ID property
```powershell
Expected 7800, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1: line 133
133:         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent.Id' | Should -Be $PID
```